pyproject.toml

```diff
@@ -31,7 +31,7 @@ dependencies = [
     "pika >= 1.2.0",
     "dataset",
     "pymongo > 3.0",
-    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@fix/issue_controller_161",
+    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@main",
```

Comment:
Let us not refer to `pce@main`, please.  It will break sdx-controller at some point.  Cab we either use a commit hash or a tag?